src/BSON/Binary.c

```diff
@@ -90,7 +90,7 @@ static bool php_phongo_binary_init_from_hash(php_phongo_binary_t *intern, HashTa
 
 /* {{{ proto void Binary::__construct(string $data, int $type)
    Construct a new BSON binary type */
-PHP_METHOD(Binary, __construct)
+static PHP_METHOD(Binary, __construct)
```

Comment:
I think this is going to break on Windows - also, as no other extension in PHP does this AFAIK, I don't think we should do this.